docs/src/content/docs/guides/authoring-content.md

```diff
@@ -361,6 +361,31 @@ A code block’s optional title can be set either with a `title="..."` attribute
   ```
   ````
 
+## Details
+
+Details (also known as “disclosures” or “accordion”) are useful to conceal information that is not immediately relevant to the user.
+The user can reveal further information with an interaction to toggle the visibility of the content.
+
+Starlight does not provide a custom Markdown syntax for details, but you can use the standard HTML `<details>` and `<summary>` elements to create them while authoring your Markdown content.
```

Comment:
I think we can describe the positive “what to do” here without mentioning what we don’t do. I do not _love_ “widget” as a word here, but it does appear to be what MDN uses, so maybe it’s OK? (And does work better than “element”, “component”, or “pattern”… any other ideas?)

```suggestion
Use the standard HTML [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) and [`<summary>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary) elements in your Markdown content to create a disclosure widget.
```